.pre-commit-config.yaml

```diff
@@ -0,0 +1,34 @@
+default_language_version:
+  python: python3
+
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.1.0
+    hooks:
+      - id: trailing-whitespace
+        exclude: .html|.svg|.css
+  - repo: https://github.com/psf/black
+    rev: 21.12b0
+    hooks:
+      - id: black
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v2.31.0
+    hooks:
+      - id: pyupgrade
+        args: [--py37-plus]
+  - repo: https://github.com/PyCQA/isort
+    rev: 5.10.1
+    hooks:
+      - id: isort
+        args: ['--profile', 'black']
+  - repo: https://github.com/nbQA-dev/nbQA
+    rev: 1.2.3
+    hooks:
+      - id: nbqa-black
+      - id: nbqa-pyupgrade
+        args: [--py37-plus]
+      - id: nbqa-isort
```

Comment:
@tlvu If you want to disable sorting of notebook imports, you just need to comment out this line. Those particular import warnings are related to an issue in CLISOPS that we need to resolve at some point (https://github.com/roocs/clisops/issues/144).

Reply:
Just spoke with the clisops team this morning. Will be opening a PR to address this on their side, hopefully this week.

Reply:
Oh great !  Next Jupyter build will hopefully pull this fix.  In the mean time I'll still undo the sort and restore the import order for homepage notebook 3 to fix current Jenkins nightly failure.